documentation/modules/deploying/proc-deploy-migrate-kraft.adoc

```diff
@@ -0,0 +1,257 @@
+      requests:
+        memory: 64Gi
+        cpu: "8"
+      limits:
+        memory: 64Gi
+        cpu: "12"    
+----
++
+NOTE: For the migration, you cannot use a node pool of nodes that share the broker and controller roles.
+
+. Apply the new `KafkaNodePool` resource to create the controllers.
++
+Errors related to using controllers in a ZooKeeper-based environment are expected.
+
+. Enable KRaft migration in the `Kafka` resource by setting the `strimzi.io/kraft` annotation to `migration`:
++
+[source,shell]
+----
+kubectl annotate kafka my-cluster strimzi.io/kraft: migration
+----
++
+.Enabling KRaft migration
+[source,yaml,subs="+attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+  namespace: my-project
+  annotations:
+    strimzi.io/kraft: migration 
+# ...
+----
++
+Applying the annotation to the `Kafka` resource configuration starts the migration.
+
+. Check the controllers have started and the brokers have rolled:
++
+[source,shell]
+----
+kubectl get pods -n my-project
+----
++
+.Output shows nodes in broker and controller node pools
+[source,shell]
+----
+NAME                     READY  STATUS   RESTARTS
+my-cluster-kafka-0       1/1    Running  0
+my-cluster-kafka-1       1/1    Running  0
+my-cluster-kafka-2       1/1    Running  0
+my-cluster-controller-3  1/1    Running  0
+my-cluster-controller-4  1/1    Running  0
+my-cluster-controller-5  1/1    Running  0
+# ...
+----
+
+. Check the status of the migration:
++
+[source,shell]
+----
+kubectl get kafka my-cluster -n my-project -w
+----
++
+.Updates to the metadata state
+[source,shell]
+----
+NAME        ...  METADATA STATE
+my-cluster  ...  Zookeeper
+my-cluster  ...  KRaftMigration
+my-cluster  ...  KRaftDualWriting
+my-cluster  ...  KRaftPostMigration
+----
++
+`METADATA STATE` shows the mechanism used to manage Kafka metadata and coordinate operations.
+At the start of the migration this is `ZooKeeper`.
++
+--
+* `ZooKeeper` is the initial state when metadata is only stored in ZooKeeper.
+* `KRaftMigration` is the state when the migration is in progress.
+The flag to enable ZooKeeper to KRaft migration (`zookeeper.metadata.migration.enable`) is added to the brokers and they are rolled to register with the controllers.
+The migration can take some time at this point depending on the number of topics and partitions in the cluster. 
+* `KRaftDualWriting` is the state when the Kafka cluster is working as a KRaft cluster, 
+but controllers are managing metadata in Kafka and ZooKeeper. 
+Brokers are rolled a second time to remove the flag to enable migration.
+* `KRaftPostMigration` is the state when KRaft mode is enabled for brokers and there is no ZooKeeper involvement. 
+Controllers are still managing in Kafka and ZooKeeper.
+You can xref:proc-deploy-migrate-kraft-rollback-{context}[roll back from this point].
+--
++
+The migration status is also represented in the `status.kafkaMetadataState` property of the `Kafka` resource. 
```

Comment:
Hmmm I would also here write something that from this standpoint the user could decide if he wants to:
1. go back to ZooKeeper (i.e., use `strimzi.io/kraft: rollback` mechanism) 
2. or continue with migration to KRaft